webcompat/static/js/lib/issues.js

```diff
@@ -411,16 +412,17 @@ issues.MainView = Backbone.View.extend({
 
       // If there are any comments, go fetch the model data
       if (this.issue.get('commentNumber') > 0) {
-        this.comments.fetch(headersBag).success(_.bind(function() {
-          this.addExistingComments();
-          this.comments.bind('add', _.bind(this.addComment, this));
-
+        var that = this;
```

Comment:
Is this needed? We should be able to use `_.bind` instead (and it seems like fetch success handler is already bound to the outer `this`.